src/base-vs-test/vsTestBase.ts

```diff
@@ -0,0 +1,37 @@
+import { describe, expect, it } from '@jest/globals';
+import { ExasolExtension } from '../api';
+import { createMockContext } from '../base/test-utils';
+
+/**
+ * This function runs shared tests for a virtual schema extension.
+ * 
+ * Call this function in your test file to test your virtual schema extension.
+ * It will assert that the parameters conform to the naming conventions.
+ * @param extensionProvider factory for the extension to test
```

Comment:
```suggestion
 * Call this function in your test file to test your virtual schema extension.
 * It will assert basic properties of your extension, e.g. having an ID, names
 * of parameters satisfy conform to the naming conventions, etc.  See the
 * descriptions of the contined test cases for details.
 * @param extensionProvider factory for the extension to test
```